### ISSUE TITLE
Quest aboard the Skybreaker in Icecrown

### ISSUE THREAD

**Issue by Floasted** [GameFreedomGG/Sindragosa#136]
There are several quest turn-ins on the Skybreaker that are not showing so you cannot complete quest to move on in the zone

**Comment by kingdeking** [comment]
Yep, Thassarian is just standing on the ship not interacting. Cant turn in the quests and unlock Ebon Blade

**Comment by MrTide** [comment]
This issue only gets fixed with server restart and its not just skybreaker it also include horde side ship we don't know what causes this problem but probably related on how many players are interacting with them.. we can't even teleport to these creatures or kill and spawn them.

**Comment by UltraNix** [comment]
I cannot reproduce it. @MrTide Can you tell me what to do?

**Comment by UltraNix** [comment]
Okay.
